decorators/ng-module/index.ts

```diff
@@ -0,0 +1,28 @@
+import importHandler from "./metadata-handlers/import";
+import daclarationHandler from "./metadata-handlers/declaration";
+import providerHandler from "./metadata-handlers/provider";
+
+export function NgModule({imports, declarations, providers, directRegister}: any) {
+    return function (target: any) {
+        let ng1ModuleIds: Array<string> = [];
+        let ng1Module = angular.module(target.name, ng1ModuleIds);
+
+        if (imports && imports.length) {
+            importHandler(ng1Module, ng1ModuleIds, imports);
```

Comment:
It's not obvious that you pass the `ng1ModuleIds` by reference. It would be better if `importHandler` returns the array, so you can pass it to `angular.module` later.